Themis/Views/Assessment/FiletreeSidebar/FiletreeSidebarView.swift

```diff
@@ -1,16 +1,31 @@
 import SwiftUI
 import SharedModels
+import DesignLibrary
 
 struct FiletreeSidebarView: View {
     @ObservedObject var cvm: CodeEditorViewModel
     @ObservedObject var assessmentVM: AssessmentViewModel
+    @Binding var repositorySelection: RepositoryType
     
     var body: some View {
         VStack(alignment: .leading) {
+            Picker(selection: $repositorySelection) {
```

Comment:
Not sure yet, if this is the optimal position for the picker. We should talk about some alternative placements